apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -67,10 +73,36 @@ export class IngestMediaItemsCliCommand extends CliCommandRunner {
 
         const generatedIds = await this.idManager.generateMany(partialPayloads.length);
 
+        const mediaLengthMap = new Map<string, number>();
+
+        for (const { filename, mimeType } of partialPayloads) {
+            const ffprobeResult = await ffprobe(`${directory}/${filename}`);
```

Comment:
Before we get too carried away, let's see if we can clean this up a bit. Let's inject a either a `FileProber` or a `MediaManager` to do this work. Let's wrap types around the result of the probe. This will allow us the option of writing integration tests if we find the test is getting slow.

Reply:
Done.